### DISCUSSION TITLE
How can I choose a specific field from the many fields that appear when I use the scanner?

### DISCUSSION THREAD

**Discussion by Gleyson1370** [AutomaApp/automa#657] [Q&A] [answered]
I need to copy the text of this field, from the "value".
How can I do with "Get Text" for me to specifically choose this field? Since it has 4 fields when I use Automa Scanner on the web page.

![image](https://user-images.githubusercontent.com/105902352/178124374-c872deff-4ff5-4dbc-bf21-f90dcc631a26.png)

**Comment by Kholid060** [comment, marked as answer]
Use the "Attribute value" block, and input `value` as the attribute name
![image](https://user-images.githubusercontent.com/22908993/178124483-a5e2cbb3-1225-4045-bf7d-5889245b53d2.png)

**Reply by Gleyson1370** [reply]
thanks a lot for the answer!
